solidity/contracts/EthGravityWrapper.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.6.6;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@uniswap/v2-periphery/contracts/interfaces/IWETH.sol";
+import "./Gravity.sol";
+
+contract EthGravityWrapper {
+	address immutable WETH_ADDRESS;
+	address immutable GRAVITY_ADDRESS;
+	uint256 constant MAX_VALUE = uint(-1);
+
+	event SendToCosmosEthEvent(
+		address indexed _sender,
+		bytes32 indexed _destination,
+		uint256 _amount
+	);
+
+	constructor(address _wethAddress, address _gravityAddress) public {
+		WETH_ADDRESS = _wethAddress;
+		GRAVITY_ADDRESS = _gravityAddress;
+ 
+		IERC20(_wethAddress).approve(_gravityAddress, MAX_VALUE);
```

Comment:
Maybe we should use `using SafeERC20 for IERC20;` on top to use safeApprove here